package-lock.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Just to note this should have no changes actually, can we revert this/remove it.